ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1605,23 +1615,25 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-longrun
     workflow: cucushift-installer-rehearse-gcp-ipi-filestore-csi
-- as: gcp-ipi-marketplace-f60
-  cron: 50 17 28 2,4,6,8,10,12 *
+- as: gcp-ipi-marketplace-mini-perm-f60
+  cron: 1 21 2 2,4,6,8,10,12 *
   steps:
     cluster_profile: gcp-qe
     env:
       COMPUTE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145
       CONTROL_PLANE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145
+      GCP_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-workers-marketplace
-- as: gcp-ipi-marketplace-f60-destructive
-  cron: 4 3 16 1,3,5,7,9,11 *
+- as: gcp-ipi-marketplace-mini-perm-f60-destructive
```

Comment:
Shall we use the same config from `gcp-ipi-mini-perm-marketplace` in 4.17+ here ?

Reply:
Oh, I inserted the keyword `mini-perm-` before `amd64|arm64|fN`. Let me update existing mini-perms tests into same manner, thanks!

Reply:
We also need to update the install workflow, use `cucushift-installer-rehearse-gcp-ipi` instead of `cucushift-installer-rehearse-gcp-ipi-workers-marketplace`, right ?